frontend/packages/console-app/src/components/nodes/NodeIPList.tsx

```diff
@@ -12,15 +12,13 @@ const NodeIPList: React.FC<NodeIPListProps> = ({ ips, expand = false }) => (
     {_.sortBy(ips, ['type']).map(
       ({ type, address }) =>
         address && (
-          <div key={`{${type}/${address}`} className="co-node-ip">
+          <DetailPropertyList key={`{${type}/${address}`}>
```

Comment:
shouldn't this be wrapped arround all ips? and not for each item

Reply:
Oops, yes, I think the `div` was here by mistake as well as it should not render when the address is not supposed to render.